src/logic/operate.js

```diff
@@ -0,0 +1,35 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  let result;
+  try {
+    const num1 = Big(+numberOne);
+    const num2 = Big(+numberTwo);
+
+    if (operation === '+') {
+      result = Big(num1.plus(num2));
+    }
+
+    if (operation === '-') {
+      result = Big(num1.minus(num2));
+    }
+
+    if (operation === 'X') {
+      result = Big(num1.times(num2));
+    }
+
+    if (operation === '÷') {
+      result = Big(num1.div(num2));
+    }
+
+    if (operation === '%') {
+      result = Big(num1.div(100));
+    }
```

Comment:
- [ ] You are creating a new instance BIg in every case after you  have wrapped numberOne and numberTwo. You can simply find the result by e.g
```
if (operation === '+') {
      result = num1.plus(num2);
 }
```